src/Components/calculator.css

```diff
@@ -0,0 +1,54 @@
+.main-calculator {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.row {
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.calculator-wrapper {
+  display: grid;
+  grid-template-columns: repeat(4, 80px);
+  grid-template-rows: auto;
+}
+
+.calculator-wrapper button {
+  border: 1px solid #d1d1d1;
+  height: 50px;
+  font-size: 15px;
+  cursor: pointer;
+  padding: 0;
+}
+
+.color {
+  background-color: #f5913e;
+}
+
+.mini {
+  grid-column: span 2;
+}
+
+input {
+  text-align: right;
+  margin: 40px 0 0 0;
+  font-size: 45px;
+}
+
```

Comment:
- [ ] With the latest changes you have added this unnecessary toggle 
![image](https://user-images.githubusercontent.com/62705947/236320120-91805ec2-8f2f-4c9a-9ec7-0273c50eebd1.png)